### ISSUE TITLE
Invalid character entity

### ISSUE THREAD

**Issue by 0x1ad2** [driftyco/ionic-cli#687]
When I use some core commands like `Ionic plugin` or `Ionic platform` I'll get the following error.

```
Error happened [Error: Invalid character entity
Line: 6
Column: 35
Char:  ]
Error: Invalid character entity
Line: 6
Column: 35
Char:
    at error (C:\Users\Dennis\AppData\Roaming\npm\node_modules\ionic\node_modules\xml2js\node_modules\sax\lib\sax.js:642:8)
    at strictFail (C:\Users\Dennis\AppData\Roaming\npm\node_modules\ionic\node_modules\xml2js\node_modules\sax\lib\sax.js:662:22)
    at Object.write (C:\Users\Dennis\AppData\Roaming\npm\node_modules\ionic\node_modules\xml2js\node_modules\sax\lib\sax.js:1336:11)
    at Parser.exports.Parser.Parser.parseString (C:\Users\Dennis\AppData\Roaming\npm\node_modules\ionic\node_modules\xml2js\lib\xml2js.js:403:31)
    at Parser.parseString (C:\Users\Dennis\AppData\Roaming\npm\node_modules\ionic\node_modules\xml2js\lib\xml2js.js:6:61)
    at Object.exports.parseString (C:\Users\Dennis\AppData\Roaming\npm\node_modules\ionic\node_modules\xml2js\lib\xml2js.js:433:19)
    at C:\Users\Dennis\AppData\Roaming\npm\node_modules\ionic\node_modules\ionic-app-lib\lib\config-xml.js:23:12
    at C:\Users\Dennis\AppData\Roaming\npm\node_modules\ionic\node_modules\ionic-app-lib\node_modules\ionic-cordova-lib\node_modules\npm\node_modules\graceful-fs\graceful-fs.js:76:16
    at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:380:3)
```
I've uninstalled Node.js and installed it again (now running Node.js v4.2.3).
Also i've uninstalled Ionic a couple of times and installed it again (globally), but I'll get the same results continusiously.

Here's my `ionic info` dump

Your system information:

Cordova CLI: 5.2.0
Gulp version:  CLI version 3.9.0
Gulp local:   Local version 3.9.0
Ionic Version: 1.1.1
Ionic CLI Version: 1.7.11
Ionic App Lib Version: 0.6.5
OS: Windows 8.1
Node Version: v4.2.3

**Comment by 0x1ad2** [comment]
Fixed it!

![yes](http://i.giphy.com/Pjr9bh5OUkgTe.gif)

Somehow during the scaffolding process (I scaffold my apps in Ionic Lab) it broke. The only insight I could give is that I accidently installed the android platform (clicked on the button in Ionic Lab) and after that pressed cancel...but that's all.

Hope you guys find out what the problem was and if not, I've got it working so I'm happy :+1: 

Not sure if I should close the issue
